src/main/java/org/hypertrace/gradle/ci/tasks/CopyReportsTask.java

```diff
@@ -37,33 +34,30 @@ public File getNamespacedOutputDir() {
 
   public CopyReportsTask() {
     this.setOutputDir(new File(getProject().getBuildDir(), "collected-reports"));
+    this.dependsOn(this.getReportingTasks(getProject()));
```

Comment:
This was just missing before - it assumed the tasks had already run whether they had or not. Caused issues on up to date checking when testing, but wouldn't have typically been seen since this is usually used on a clean CI build after tests have run.